web/client/plugins/print/index.js

```diff
@@ -119,6 +119,7 @@ const Font = connect((state) => ({
     italic: state.print && state.print.spec && state.print.spec.italic
 }), {
     onChangeFamily: setPrintParameter.bind(null, 'fontFamily'),
+    onChangeName: setPrintParameter.bind(null, 'fontName'),
```

Comment:
fontName is not used, I think. You still use only fontFamily in PrintUtils. Maybe we don't need a duplicated property at all